frontend/src/components/payment/paypal-payment-dialog.js

```diff
@@ -6,25 +6,38 @@ import Dialog, {
   DialogContentText,
   DialogTitle
 } from 'material-ui/Dialog'
+import CheckBox from 'material-ui/CheckBox'
 
 class PaypalPaymentDialog extends Component {
+  constructor (props) {
+    super(props)
+    this.agreeTermsPaypal = this.agreeTermsPaypal.bind(this)
```

Comment:
You don't need this if you use arrow functions on `agreeTermsPaypal` method inside this component